### PR TITLE
Fix missing option loads from env

### PR DESCRIPTION
**Pull request by naoyam**
https://github.com/NVIDIA/Fuser/pull/498#discussion_r1238400240

### CODE REVIEW

**Review by zasdfgbnm — APPROVED** [review]

File: csrc/options.h [hunk 1]
```diff
@@ -110,7 +110,7 @@ enum class DisableOption {
 template <typename OptionEnum>
 class Options {
  public:
-  Options() = default;
+  Options() : options_(getOptionsFromEnv()) {}
```
[the remark this answers is not in the document]
Comment (jjsjann123) [reply csrc/options.h:113]: note that I'm not saying this is wrong... but we might need to update the scripts :yum:

[the remark this answers is not in the document]
Comment (naoyam) [reply csrc/options.h:113]: I was still running the torchscript tests, and you're right, they failed. Let me try again.

[the remark this answers is not in the document]
Comment (jjsjann123) [reply csrc/options.h:113]: I think we can just move these earlier than the import https://github.com/NVIDIA/Fuser/blob/main/python_tests/test_torchscript.py#L62-L73

Note that `import torch` would already dllopen nvfuser and initialize all static variables. So before line 16, maybe.
